src/constants/actionTypes.js

```diff
@@ -0,0 +1,5 @@
+import keyMirror from 'keymirror';
+
+export default keyMirror({
```

Comment:
so, obviously ymmv, but I liked idea of namespacing my actions a little deeper, so I had to use something a little different than keyMirror, because it doesn't support nesting (see: https://github.com/smashingboxes/univ-denv_fe/blob/master/src/constants/ActionTypes.js and https://github.com/smashingboxes/univ-denv_fe/blob/master/src/constants/helpers/recursiveKeyMirror.js)

Reply:
That's pretty interesting. Oh maybe you should publish an NPM package for it! 💯 
I'm going to leave it out of the boilerplate for now though

Reply:
Or better yet, make a PR to keymirror! Open Source ftw!